src/state/middlewares/socket-io/chatlist.js

```diff
@@ -353,6 +355,18 @@ export default ( { io, timeout = 1000, customerDisconnectTimeout = 90000, custom
 		.catch( e => debug( 'failed to remove operator from chat', e.message ) )
 	}, chat_id => debug( 'chat.leave without existing chat', chat_id ) )( action.chat_id, action.user )
 
+	const handleCustomerBlock = action => whenChatExists( ( chat  ) => {
```

Comment:
That probably was because we are not using `eslines`, [yet](https://github.com/Automattic/happychat-service/pull/122/files) :)